pkg/cdi/container-edits.go

```diff
@@ -280,6 +295,20 @@ func (m *Mount) Validate() error {
 	return nil
 }
 
+// IntelRdt a wrapper used for validating IntelRdt configuration.
+type IntelRdt struct {
```

Comment:
nit: Is this type used?

Reply:
Argh, unused indeed, stale type from some early implementation. Well spotted, thanks. Now removed